src/formatters/codeFrameFormatter.ts

```diff
@@ -67,7 +67,7 @@ export class Formatter extends AbstractFormatter {
             }
 
             let failureString = failure.getFailure();
-            failureString = chalk.red(failureString);
+            failureString = failure.getRuleSeverity() === "warning" ? chalk.yellow(failureString) : chalk.red(failureString);
```

Comment:
can you please add a test for this in `test/formatters/codeFrameFormatterTests.ts`?

Reply:
How can I add another commit to this PR?  Because it was created by editing a file in the web browser I just see `merge 1 commit into palantir:master from jez9999:patch-2` rather than merging from a proper fork.